torch/csrc/jit/runtime/argument_spec.h

```diff
@@ -271,6 +274,10 @@ struct CompleteArgumentSpec {
         }
       }
       // each POD has a running tally of all dimensions including its own
+      TORCH_CHECK(
```

Comment:
I hope we won't hit this limit. Practically, it could probably only happen with code-generated models with lots of arguments.